spark/src/main/scala/org/apache/mahout/drivers/MahoutSparkDriver.scala

```diff
@@ -46,7 +45,7 @@ import org.apache.mahout.sparkbindings._
   *       opts = opts ++ SomeOptions
   *       note("\nAlgorithm control options:")
   *       opt[Int]("maxThings") abbr ("mt") action { (x, options) =>
-  *         options + ("maxThings" -> x) ...
+  *         options + ("maxThings" ->  x) ...
```

Comment:
an extra space didn't seem worth the commit history. The original substantive change was a missing ">" typo if you care.